### ISSUE TITLE
Expression Bodied Members in MarkMessagePropertiesInternalSealed Branch

### ISSUE THREAD

**Issue by smudge202** [sblackler/compose-remoting#38]
@sblackler , you've used expression bodied members as opposed to static. Why?

```c#
internal static string TypeName => "TypeName";
```

Why not

```c#
internal static string TypeName = "TypeName";
```

**Comment by sblackler** [comment]
Resharper suggested that I could do that instead of the property. I agreed, so tidied it up.  Can change if you really want, but I hope this class goes as part of discussions in #28 #10

**Comment by smudge202** [comment]
Not sure if we lost some code throughout the day but these currently show as:

```c#
public static string CreatedAtUtc
{
    get { return "CreatedAtUtc"; }
}
```

Can be changed to:

```c#
public const string CreatedAtUtc = "CreatedAtUtc"
```

**Comment by sblackler** [comment]
Resolved as part of PR #69
